### PR TITLE
Rename master branch to stable

### LINKED ISSUE #2026: [Housekeeping] Rename `master` branch to `stable`
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request

To reflect our release management strategies the `master` branch should be renamed ➡️  `stable`.

When renaming the branch CI/CD stuff will break and must be updated.

- [ ] Rename branch
- [ ] Fix CI/CD

## Have you considered any alternatives?

Other names that are _not_ similar to `main`.

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #2026

## What is the new behavior?
Across the code, any reference to `master` branch has been renamed to `stable` to better reflect our release workflow.

Right before merging this, we should rename the `master` branch to `stable` here on github. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] ~~Make sure you have implemented tests following the guidelines in: "[The good: Test]~~(https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] ~~Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).~~

### Review  
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] ~~Request that the changes are code-reviewed~~ 
- [x] ~~Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)~~

When the pull request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ This consists of code-reviewing your own pull request, following guidelines foun
 
 > Confirmation from UXers that the implementation complies with what was decided during UX Refinement.
 
-If the implementation of the issue resulted in visual changes - then it has to be UX reviewed. This is to make sure that the introduced changes align with the design system. We strive to have every issue that introduces visual changes UX reviewed. If this is not done and the changes are merged to master, we might recieve issues about components not being properly aligned with the design system anymore from the users of Kirby.
+If the implementation of the issue resulted in visual changes - then it has to be UX reviewed. This is to make sure that the introduced changes align with the design system. We strive to have every issue that introduces visual changes UX reviewed. If this is not done and the changes are merged to stable, we might recieve issues about components not being properly aligned with the design system anymore from the users of Kirby.
```
Comment (jkaltoft) [inline .github/CONTRIBUTING.md:84]:
```suggestion
If the implementation of the issue resulted in visual changes - then it has to be UX reviewed. This is to make sure that the introduced changes align with the design system. We strive to have every issue that introduces visual changes UX reviewed. If this is not done and the changes are merged to `stable`, we might recieve issues about components not being properly aligned with the design system anymore from the users of Kirby.
```

**Review by jkaltoft — DISMISSED** [review]
image


I would expect the Github workflows to run exactly as before. 

I've added a suggestion, but I think it's optional.

**Comment by MadsBuchmann** [comment]
Should anything be done in local repositories when this change is merged 👀 ?

**Comment by RasmusKjeldgaard** [comment]
> Should anything be done in local repositories when this change is merged 👀 ?

Yes! I'll do a quick writeup of [the details](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes) later, in a discussion post!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (RasmusKjeldgaard) [reply .github/CONTRIBUTING.md]: Good idea, added backticks 👍

**Review by jkaltoft — APPROVED** [review]

**Pull request merged**